Stormshield/sns/_meta/manifest.yml

```
@@ -0,0 +1,4 @@
+uuid: 79029ef9-e5d3-44f3-b70f-fd3b54ba1fe4
+name: Stormshield Network Security
+slug: sns
```

Isn’t that too short for a slug? 

```suggestion
slug: stormshield-network-security
```